layouts/_default/list.html

```diff
@@ -41,7 +41,7 @@ <h2>
       <hr>
       {{ if .Page.Lastmod }}
       <div class="last-modified">
-        Last modified {{ .Page.Lastmod.Format "January 2, 2006" }}
```

Comment:
Oh... I didn't know how to solve this problem when I implemented these links in a previous job. Thank you!